actions/chains/bgp_prefix_exceeded_chain.yaml

```diff
@@ -0,0 +1,75 @@
+---
+  chain:
+      -
+        name: "show_bgp_neighbours"
```

Comment:
So.......not trying to start a war here, but in this case I would prefer that we stick to the naming conventions used in the NAPALM project itself. Throughout the project, the word `neighbor` is used.

Please direct all complaints to @dbarrosop - I'm just the messenger :)